src/views/hardware_view.rs

```diff
@@ -703,7 +705,10 @@ mod test {
 
         // Test case: No function selected
         let result = filter_options(&options, None);
-        assert_eq!(result, vec![PinFunction::Input(None), PinFunction::Output(None)]);
+        assert_eq!(
+            result,
+            vec![PinFunction::Input(None), PinFunction::Output(None)]
+        );
```

Comment:
these are all new formatting change, right?

Reply:
not sure, i haven't staged this changes